### ISSUE TITLE
no clone

### ISSUE THREAD

**Issue by benitogf** [FamousArchives/examples#25]
Permission denied (publickey).
fatal: Could not read from remote repository.

**Comment by michaelobriena** [comment]
@b3n0n it looks like you haven't included your key in github.  Try looking at https://help.github.com/articles/error-permission-denied-publickeyn and maybe https://help.github.com/articles/generating-ssh-keys if you need to generate new keys.

**Comment by fczuardi** [comment]
The readme uses git@github... I manage to clone it using the http version:

```
git clone https://github.com/Famous/examples.git --recursive
```

**Comment by benitogf** [comment]
no sorry just tested 

ssh -T git@github.com 

all good with the ssh key but still

Permission denied (publickey).
fatal: Could not read from remote repository.

**Comment by fczuardi** [comment]
~~here is a pull request https://github.com/Famous/examples/pull/26~~ forget that, the issue remains

**Comment by benitogf** [comment]
I tried with http too, nothing 

sudo git clone https://github.com/Famous/examples.git --recursive famous

Cloning into 'famous'...
remote: Reusing existing pack: 1639, done.
remote: Counting objects: 7, done.
remote: Compressing objects: 100% (7/7), done.
remote: Total 1646 (delta 0), reused 0 (delta 0)
Receiving objects: 100% (1646/1646), 1.57 MiB | 38.00 KiB/s, done.
Resolving deltas: 100% (796/796), done.
Checking connectivity... done
Submodule 'src/famous' (git@github.com:Famous/famous.git) registered for path 'src/famous'
Cloning into 'src/famous'...
Warning: Permanently added the RSA host key for IP address '192.30.252.130' to the list of known hosts.
Permission denied (publickey).
fatal: Could not read from remote repository.

**Comment by fczuardi** [comment]
strange, I got the same problem as you the first time, but using http ~~worked for me~~(nope, it didn't)

(or maybe it didn't because the index.html file gives me require.js errors when I try to open in the browser, but that's probably a different bug)

**Comment by benitogf** [comment]
That means that you cloned the examples but not the src files, you are having the same error

Free the src folder! :8ball: :-1:

**Comment by fczuardi** [comment]
you are totally right @b3n0n ! my bad :(

**Comment by fczuardi** [comment]
The workaround for me was to clone https://github.com/Famous/famous and them copying the app/lib/famous folder into src/famous

**Comment by meheller** [comment]
I had to clone from the GitHub 2 desktop app.

_Martin Heller_
+1-978-464-1831 (Google Voice)

On Thu, Apr 24, 2014 at 8:06 PM, Fabricio C Zuardi <notifications@github.com

**Comment by benitogf** [comment]
@fczuardi you used the yeoman generator? I tried the submodule init and got the same error

**Comment by fczuardi** [comment]
yep, yeoman generator, :P

**Comment by MylesBorins** [comment]
If you are using the yeoman generator there are no submodules

**Comment by mrqc** [comment]
got the same problem. I forked the famous repo to my own, ALSO every submodule and the checked it out and ALSO checked every submodule out in the correct folder

**Comment by MylesBorins** [comment]
This is solved by merging either #29 or #31 , closing for now
